### DISCUSSION TITLE
Unomi API ?

### DISCUSSION THREAD

**Discussion by tomdavidson** [Tracardi/tracardi#623] [Q&A]
Does TRACARDI also offer a full Unomi API that can be integrated with directly?

**Comment by atompie** [comment]
@tomdavidson It is on our integration list. If you would like to use TRACARDI with Unomi let us know. We will move the priority of it and it will be available very soon. Of course, if you do not use Unomi right now and looking for a CDP, TRACARDI can also serve as CDP.
